core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -54,17 +55,18 @@ contract PricelessPositionManager is FeePayer {
 
     // Unique identifier for DVM price feed ticker.
     bytes32 public priceIdentifer;
-    // Time that this contract expires.
+    // Time that this contract expires. Should not change post-construction unless a emergency shutdown occurs.
```

Comment:
@mrice32 @chrismaree I tried suggesting a change (for you to commit) here, but accidentally just committed my suggested change